hold_duration:      10.0  # Duration to wait inside docking bay
goal_tolerance_pos: 0.70 # Distance tolerance for arriving at goal pose
goal_tolerance_ang: 0.30 # Angular tolerance for arriving at goal yaw

#pix_threshold: 30 # (px) Threshold for centering the image
pix_offset:     10 # (px) Offset center x position in image

n_entry_wps:     5 # Number of waypoints to split dock bay entry into
entry_duration: 20 # Time to enter into dock bay
